itou/approvals/management/commands/fill_approval_origin_fields.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from django.core.management.base import BaseCommand
+
+from itou.approvals.models import Approval
+from itou.job_applications.models import JobApplicationWorkflow
+
+
+BATCH_SIZE = 1000
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        objects_to_migrate = Approval.objects.filter(
+            origin_sender_kind=None,
+            jobapplication__state=JobApplicationWorkflow.STATE_ACCEPTED,
+        ).distinct()
+        total_objects = objects_to_migrate.count()
+        print(f"Before: {total_objects}")
+
+        batch = []
+        for approval in objects_to_migrate.iterator():
```

Comment:
Si on veux être encore plus doux, je pense qu'on peux se limiter à seulement récupérer la PK et pas tout les champs :
```suggestion
        for approval in objects_to_migrate.only("pk").iterator():
```